src/gui/dialog/SettingsDialog.cpp

```diff
@@ -202,6 +208,7 @@ void SettingsDialog::load() {
     loadCheckbox("cbHideVerticalScrollbar", settings->getScrollbarHideType() & SCROLLBAR_HIDE_VERTICAL);
     loadCheckbox("cbDisableScrollbarFadeout", settings->isScrollbarFadeoutDisabled());
     loadCheckbox("cbTouchWorkaround", settings->isTouchWorkaround());
+    loadCheckbox("cbIgnoreFirstStylusEvents", settings->getIgnoredStylusEvents() != 0);  // 0 means disabled, >0 enabled
```

Comment:
Repeated use of `settings->getIgnoredStylusEvents() != 0` is error-prone. Please store it in a local variable `const bool ignoreStylusEventsEnabled = ...` and reuse on line 227.